go.mod

```diff
@@ -424,7 +424,6 @@ require (
 	github.com/Microsoft/go-winio v0.6.1 // indirect
 	github.com/ProtonMail/go-crypto v0.0.0-20230828082145-3c4c8a2d2371 // @grafana/plugins-platform-backend
 	github.com/RoaringBitmap/roaring v0.9.4 // indirect
-	github.com/acomagu/bufpipe v1.0.3 // indirect
```

Comment:
@papagian i'm not sure if this is going to help, but here is what i did for this commit:
* `go mod tidy` (no changes)
* `make run` (build failed)
* `go mod tidy` again (changes to go.mod and go.sum)
* `make run` (build pass)